recipes/y-py/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "y-py" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/yjs/y-crdt/archive/v{{ version }}.tar.gz
+  sha256: ae67b4c261277ea94d9435a0581696b24b1e191f5c5b82059147d6409974ca37
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install ./y-py -vv
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+  host:
+    - maturin
+    - pip
+    - python
+  run:
+    - python
+    - libgcc-ng  # [linux]
```

Comment:
add `{{ compiler('gcc') }}` up in `build` and this will get put in  automatically